README.md

```diff
@@ -101,14 +105,16 @@ The Starlink router also exposes a gRPC service, on ports 9000 (HTTP/2.0) and 90
 Initialization of the container can be performed with the following command:
 
 ```
-docker run -d --name='starlink-grpc-tools' -e INFLUXDB_HOST={InfluxDB Hostname} \
+docker run -d -t --name='starlink-grpc-tools' -e INFLUXDB_HOST={InfluxDB Hostname} \
     -e INFLUXDB_PORT={Port, 8086 usually} \
     -e INFLUXDB_USER={Optional, InfluxDB Username} \
     -e INFLUXDB_PWD={Optional, InfluxDB Password} \
     -e INFLUXDB_DB={Pre-created DB name, starlinkstats works well} \
-    neurocis/starlink-grpc-tools dishStatusInflux.py -v
+    neurocis/starlink-grpc-tools dishStatusInflux.py -v -t 30
 ```
 
-`dishStatusInflux.py -v` is optional and will run same but not -verbose, or you can replace it with one of the other scripts if you wish to run that instead. There is also an `GrafanaDashboard - Starlink Statistics.json` which can be imported to get some charts like:
+The `-t` option to `docker run` will prevent Python from buffering the script's standard output and can be omitted if you don't care about seeing the verbose output in the container logs as soon as it is printed.
```

Comment:
While testing with docker, I noticed there wasn't any output until the script terminated, unless I told it to allocate a TTY.